### ISSUE TITLE
Sui Doc Bug or Feature Request

### ISSUE THREAD

**Issue by mavroudisv** [MystenLabs/sui#2585]
This is not purely a doc issue but it can be easily resolved by updating the docs.

> What are the existing URLs containing the issue, if any?

https://docs.sui.io/build/fullnode?utm_source=medium&utm_medium=social&utm_campaign=fullnode

> How would you describe the issue? (What needs changing?)

The docs mention port 9000 as being the RPC port (this is indeed correct)

Hence, the docs guide the user to use the 'local' option on the explorer's page:

> Click the Devnet button in the top right-hand corner of the Explorer and select the Local network from the drop-down menu.

However, the devnet explorer is looking for an RPC on port 5001.

`r,i.Local,"http://127.0.0.1:5001")` in https://explorer.devnet.sui.io/static/js/main.d2181072.js

and thus fails to load any transactions.

> What precisely must they do with it? (Configure a new service? Process some output?)

Either update the docs to ask the user to provide a custom RPC endpoint pointing to the correct port: `http://127.0.0.1:9000`

or update the fullnode binary to bind to port 5001.

**Comment by gdanezis** [comment]
Thanks Vasilis -- triage under bug!

**Comment by bmwill** [comment]
Thanks for identifying this issue. The reason that explorer is currently set to 5001 is because that was the default port that the Gateway service runs on. We're actually in the process of folding in most of gateway's functionality into fullnodes and this was something that we knew we needed to update but apparently missed before announcing fullnode support. This has been addressed in https://github.com/MystenLabs/sui/pull/2592 and an update to the explorer should be rolled out soon with the fix.
